hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/volume/DbVolume.java

```diff
@@ -55,11 +55,10 @@ protected DbVolume(Builder b) throws IOException {
     super(b);
 
     this.hddsDbStorePathMap = new HashMap<>();
-    if (!b.getFailedVolume()) {
-      LOG.info("Creating DbVolume: {} of storage type: {}, {}",
-              getStorageDir(), b.getStorageType(),
-              getCurrentUsage());
 
+    LOG.info("Creating DbVolume: {}", getReport());
```

Comment:
Can you move this LOG message into the following if statement?

Reply:
I think logging failed volume provides information.

Reply:
Then please remove the "Creating" from the log, show "DbVolume: {}" only.

Reply:
Then how about log the message at the end of the function, and remove the id null check?